test/types/index.ts

```diff
@@ -324,19 +324,23 @@ server.setSchemaCompiler(function (schema: object) {
 
 server.addSchema({})
 
-server.addContentTypeParser('foo/bar', {}, (req, done) => {
```

Comment:
Could you keep this too?

Reply:
This conflicts with your docs. It says:
```
As you can see, now the function signature is (req, body, done) instead of (req, done)
```

Reply:
This is allowed. The body parameter is in the signature only if `returnAs` is specified

Reply:
https://github.com/fastify/fastify/pull/1162/files#r217922743